IBM/carbon-addons-iot-react
# IconSwitcher visual defects

Closes #1127

**Summary**

Fixed the visual defects around focus, active and hover.

**Change List (commits, features, bugs, etc)**

Added style to icon-switch.scss 
- to override inherited focus, hover, active states for svg
- doubled the z-index for tooltip on hover so that focused tooltip does not (partially) block it.

**Acceptance Test (how to verify the PR)**

- Click on IconSwitch in storybook
- Choose "unselected"
- Click and focus (using Tab) on icon
- Now move cursor away
- Icon svg should now change back to gray (as opposed to white)
- Now click on "example-used-in-contentswitcher"
- Click and focus (using Tab) on an icon
- Hover over the other icon
- Hovered tooltip should stay on top over the focused tooltip
